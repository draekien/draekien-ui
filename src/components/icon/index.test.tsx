/** @jsxRuntime classic */
/** @jsx jsx */
import { render } from '@testing-library/react';
import { jsx } from 'theme-ui';
import Icon from '.';

describe('Icon component', () => {
  test('default icon matches snapshot', () => {
    const component = render(<Icon name="accessibility" />);

    expect(component.container).toMatchSnapshot();
  });
});
