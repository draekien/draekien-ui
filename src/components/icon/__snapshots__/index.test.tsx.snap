// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component default icon matches snapshot 1`] = `
<div>
  <i
    class="material-icons css-eyhhi6"
  >
    accessibility
  </i>
</div>
`;
